app/server/templates/projects.html

```diff
@@ -3,151 +3,10 @@
-        <button class="button is-danger" @click="deleteProject()">Delete</button>
-        <button class="button" @click="isDelete=!isDelete">Cancel</button>
-      </footer>
-    </div>
-  </div>
-
-  <section class="hero">
-    <div class="container">
-      <div class="columns">
-        <div class="column is-10 is-offset-1">
-      <div class="card events-card">
-        <header class="card-header">
-          <p class="card-header-title">
-            [[ items.length ]] Projects
-          </p>
-          <div class="field card-header-icon">
-            <div class="control">
-              <div class="select">
-                <select v-model="selected">
-                  <option selected>All Project</option>
-                  <option>Text Classification</option>
-                  <option>Sequence Labeling</option>
-                  <option>Seq2seq</option>
-                </select>
-              </div>
-            </div>
-          </div>
-        </header>
-        <div class="card-table">
-          <div class="content">
-            <table class="table is-fullwidth">
-              <tbody>
-                <tr v-for="project in selectedProjects">
-                  <td class="pl15r">
-                    <div class="thumbnail-wrapper is-vertical">
-                      <img class="project-thumbnail" v-bind:src="project.image">
-                    </div>
-                    <div class="dataset-item__main is-vertical">
-                      <div class="dataset-item__main-title">
-                        <div class="dataset-item__main-title-link dataset-item__link">
-                          <a v-bind:href="'/projects/' + project.id" class="has-text-black">[[ project.name ]]</a>
-                        </div>
-                      </div>
-                      <div class="dataset-item__main-subtitle">
-                        [[ project.description ]]
-                      </div>
-                      <div class="dataset-item__main-info">
-                        <span class="dataset-item__main-update">updated <span>[[ getDaysAgo(project.updated_at) ]] days ago</span></span></div>
-                    </div>
-                  </td>
-                  <td class="is-vertical"><span class="tag is-normal">[[ project.project_type ]]</span></td>
-                  {% if user.is_superuser %}
-                  <td class="is-vertical"><a v-bind:href="'/projects/' + project.id + '/docs'">Edit</a></td>
-                  <td class="is-vertical"><a class="has-text-danger" @click="setProject(project)">Delete</a></td>
-                  {% endif %}
-                </tr>
-              </tbody>
-            </table>
-          </div>
-        </div>
-      </div>
-      </div>
-      </div>
-    </div>
-  </section>
-
-</div>
+<div id="projects_root"></div>
 {% endblock %}
 {% block footer %}
+{{ user.get_username | json_script:"django.username" }}
+{{ user.is_superuser | json_script:"django.is_superuser" }}
```

Comment:
I think Django and Vue are somewhat coupled in these lines.
Is the Me API I submitted in https://github.com/chakki-works/doccano/pull/151 useful for decoupling?

Reply:
Yes! This would be a great next step refactor. We can either first merge the Me API and then I can update this PR or we can first merge the PR, then merge Me API and refactor in the future to further decrease the coupling.

Reply:
OK, first merge the Me API and update this PR.

Reply:
Excellent. Please ping this thread when the Me API is merged and I'll make the required changes.

Reply:
Merged.

Reply:
Refactored to use the /me API in https://github.com/chakki-works/doccano/pull/148/commits/72d91c6188f91e5bfe1a873431b8868353bd5677.